prime-router/metadata/hl7_mapping/ORU_R01/base/order-observation.yml

```diff
@@ -33,7 +33,7 @@ elements:
 
   - name: observation-result-with-aoe
     # Grab only the AOE observations from ServiceRequest.supportingInfo NOT associated with a specimen
-    resource: '%resource.result.resolve() | %service.supportingInfo.resolve().where(specimen.exists().not())'
```

Comment:
(not intending to merge this) @mkalish I commented out the second piece of this, and integration tests remained passing. Was this added because we needed it, or was it more preemptive for future possible data?

Reply:
I don't think there is an integration test that would cover this, but it's future proofing so that we still map AOE questions not associated with an a specimen.

Reply:
Ok yea that makes sense, there's an issue then with the observation value in those cases, but hopefully we can make the resource/parent/outerResource changes we talked about to help enable this to work before those cases come up.